src/App.js

```diff
@@ -0,0 +1,57 @@
+// import logo from './logo.svg';
+import './App.css';
+
+function App() {
+
+
+  return (
+    <div className="App">
+      <div className='Section1'>
+        <nav className='App-header'>
+          <img src='/images/ironhack-logo-xs.png' alt='logo'></img>
+
+          <img className='menuLogo' src='/images/menu-top-xs.png' alt='menu'></img>
+        </nav>
+
+        <div className='divText'>
+        <h1>Say Hello to <br></br> ReacJs</h1>
+        <p>You will learn how to use <br></br>the most popular frontend library, <br></br>and become a super Ninja developer</p>
+
+        <button><strong>Awesome!</strong></button>
+        </div>  
+
+      </div>
+
+      <div className='cards'>
```

Comment:
in this case when data is added and there is no server like static pages you can easely mock some object in order to espect that from seerver later as a front end you will like it as a backend they gonna love it 

```
const dataCardArray = [
{
title: "budooo",
image: "image url.com",
text : "like pro",
}
]